packages/ion_identity_client/lib/src/core/types/ion_exception.dart

```diff
@@ -27,7 +27,11 @@ class PasskeyValidationException extends IONIdentityException {
 }
 
 class TwoFARequiredException extends IONIdentityException {
-  const TwoFARequiredException() : super('Two-factor authentication is required');
+  const TwoFARequiredException(
+    this.twoFAOptionsCount,
+  ) : super('Two-factor authentication is required');
```

Comment:
l10n for this text?

Reply:
This package is outside of the app's scope. Basically, I suppose we don't need this message at all, it's just for debug purpose.